packages/eas-build-job/src/android.ts

```diff
@@ -134,16 +138,23 @@ export const JobSchema = Joi.object({
   type: Joi.string()
     .valid(...Object.values(Workflow))
     .required(),
+  triggeredBy: Joi.string().valid(...Object.values(BuildTrigger)),
```

Comment:
Maybe default to EAS CLI so you can make `triggeredBy?: BuildTrigger;` non-optional.